image/image_linux.go

```diff
@@ -710,7 +710,7 @@ func manifestFromLocalSnaps(snaps localSnapRefs, opts *Options) (map[string]snap
 	return seedManifest, nil
 }
 
-func selectAssertionMaxFormats(tsto *tooling.ToolingStore, sysSn *seedwriter.SeedSnap) error {
+func selectAssertionMaxFormats(tsto *tooling.ToolingStore, model *asserts.Model, sysSn, kSn *seedwriter.SeedSnap) error {
```

Comment:
Maybe it's because I'm not familiar with this code but the meaning of `kSn` wasn't very clear on its own. Maybe we could extend it to `kernelSn` or `kernelSnap`?

Reply:
went for kernSn